src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -294,6 +297,22 @@ private FolioExecutionContext prepareContextForTenant(String tenantId, FolioExec
     return new DefaultFolioExecutionContext(folioModuleMetadata, context.getOkapiHeaders());
   }
 
+  public void deleteShadowUsers(UUID userId) {
```

Comment:
how will this method behave when one out of three shadow users return an error during deletion?

Reply:
Olamide :  i have tested  some cases , and mod-users deleteByQuery method always returns success response. 
and deleteUsers() is been called inside @transactional annotation so incase of error . it should be rolled back. thanks